ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -643,6 +661,12 @@ def run_data_profiler(self, table: str, schema: str) -> TableProfile:
         logger.debug(f"Finished profiling {dataset_name}")
         return profile
 
+    def _build_database_state(self, schema_fqdn: str) -> [EntityReference]:
+        database = self.metadata.get_by_name(
+            entity=Database, fqdn=schema_fqdn, fields=["tables"]
+        )
+        return database.tables.__root__
```

Comment:
Something looks odd here in terms of python semantics
Usually __xyz__ refers to an internal method that is not accessed directly. 

I suppose the jsonchema to pydantic model does not generate iterators as mentioned over [here](https://pydantic-docs.helpmanual.io/usage/models/#custom-root-types)

Reply:
@mithmatt are you talking about __root__ element reference or accessing database.tables?

Reply:
Edited comment (didn't realize that my variable got formatted and dropped the `__` prefix and suffix)

Yes, I meant `__root__` element reference